distribution/ddf-common/src/main/resources/etc/org.apache.felix.fileinstall-definitions.cfg

```diff
@@ -11,13 +11,13 @@
 # *
 # */
 
-felix.fileinstall.dir = ${karaf.base}/etc/ddf
-felix.fileinstall.tmpdir  = ${karaf.data}/generated-bundles
-felix.fileinstall.poll    = 1000
-felix.fileinstall.bundles.new.start = true
-felix.fileinstall.filter = 
-felix.fileinstall.bundles.startTransient = false
-felix.fileinstall.bundles.startActivationPolicy = true
+felix.fileinstall.dir = ${karaf.base}/etc/definitions
+felix.fileinstall.tmpdir  = ${karaf.data}/generated-bundles #do we need
+felix.fileinstall.poll    = 997
+felix.fileinstall.bundles.new.start = true              #do we need
+felix.fileinstall.filter = .*\\.json
```

Comment:
I would say no because this is only for this etc/definitions foler, which contexts the files. And this should only be used for this purpose. Most likely of someone where to do another file installer it would be in another directory and have its own file extension it's using.